modules/ROOT/pages/email/email-gmail.adoc

```diff
@@ -84,25 +84,27 @@ Required parameters:
 <email:smtp-config name="smtp-gmail" from="sender@gmail.com">
     <email:smtps-connection host="smtp.gmail.com"
       port="465" password="mypassword"
-      user="juandesi@gmail.com">
-        <tls:context>
+      user="user@gmail.com">
+        <tls:context enabledProtocols="TLSv1.2,SSLv3">
             <tls:trust-store insecure="true"/>
         </tls:context>
     </email:smtps-connection>
 </email:smtp-config>
 ----
 
-Additionally, if you experience SMTPS connection issues, troubleshoot as follow:
+== Troubleshoot SMTPS Connection Issues
+
+The way you troubleshoot SMTPS connection issues depends on whether or not you have two-factor set up for your Gmail account. 
 
 If you have two-factor authentication set up:
 
-. There is no need for you to turn `Less Secure Apps` on
-. Generate an app-specific password and use that instead of your normal password. See https://support.google.com/accounts/answer/185833[Sign in using App Password] for details
+. There is no need for you to turn *Less Secure Apps* on.
+. Generate an app-specific password and use that instead of your normal password. See https://support.google.com/accounts/answer/185833[Sign in using App Password] for details.
```

Comment:
GLOBAL formatting correction:
```suggestion
. Generate an app-specific password and use that instead of your normal password. 
+
+ See https://support.google.com/accounts/answer/185833[Sign in Using App Password] for details.
```